pytest_structlog.py

```diff
@@ -97,7 +100,9 @@ def log(monkeypatch, request):
     monkeypatch.setattr("structlog.configure", no_op)
     monkeypatch.setattr("structlog.configure_once", no_op)
     request.node.structlog_events = cap.events
+    clear_contextvars()
     yield cap
+    clear_contextvars()
```

Comment:
Why twice though? If I understand correctly it should be sufficient to clear just once (either before or after)

Reply:
In the unlikely case that someone doesn't use the `log` fixture but accesses the contextvars.